src/integrationTest/java/uk/gov/hmcts/reform/authorisation/TestApplication.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.reform.authorisation;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+@SuppressWarnings("HideUtilityClassConstructor") // Spring needs a constructor, its not a utility class
+public class TestApplication {
+
+    public static void main(final String[] args) {
+        SpringApplication.run(TestApplication.class, args);
```

Comment:
don't think this is needed?
Here's a simple set of tests that use most of the clients:
https://github.com/hmcts/ccd-client/tree/master/src/functionalTest/java/uk/gov/hmcts/reform/ccd/client

Reply:
true. had to add couple more annotations though